hack/common.sh

```diff
@@ -21,6 +21,8 @@ MANIFEST_TEMPLATES_OUT_DIR=$OUT_DIR/templates/manifests
 PYTHON_CLIENT_OUT_DIR=$OUT_DIR/client-python
 ARCHITECTURE=$(uname -m)
 
+kubectl() { cluster-up/kubectl.sh "$@"; }
```

Comment:
Can you move all changes from here into a separate shell script? This script can look like this:

```
set -ex pipefail

source hack/common.sh
source cluster-up/cluster/$KUBEVIRT_PROVIDER/provider.sh
source hack/config.sh

function find_node_names() {

}

function find_pull_command() {

}
```

`cluster_deploy.sh` and `cluster_build.sh` can source this file instead. 

You can then do the follwoing:
 * use `_kubectl` everywhere instead of introducing `kubectl`
 * keep kubevirtci related code completely out of common
 * import this new script in other scripts which require kubevirtci

Reply:
Right, and looks much better now